### DISCUSSION TITLE
Start chromedriver inside of php

### DISCUSSION THREAD

**Discussion by mmehdizadeh** [php-webdriver/php-webdriver#923] [Q&A]
Hi guys, Is there anyway to start server inside of php ?
Something like this:

`exec('chromedriver --port=4444');`
        `$serverUrl = 'http://localhost:9515';`
        `$driver = RemoteWebDriver::create($serverUrl, DesiredCapabilities::chrome());`

This code not working on my case.

**Comment by andrewnicols** [comment]
This isn't really a php-webdriver problem. The `exec` will wait for the binary to finish (which it won't). You need to fork. You'll probably also need to wait for the init to finish before you start WebDriver.
